src/Microsoft.NetCore.Analyzers/UnitTests/Security/UseXmlReaderForSchemaReadTests.cs

```diff
@@ -90,14 +88,22 @@ public XmlSchema Test
 }");
         }
 
-        protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
+        private static async Task VerifyCSharp(string source, params DiagnosticResult[] expected)
         {
-            return new UseXmlReaderForSchemaRead();
-        }
+            var csharpTest = new VerifyCS.Test
+            {
+                TestCode = source,
+                TestBehaviors = TestBehaviors.SkipGeneratedCodeCheck
+            };
```

Comment:
📝 This is just `CSharpSecurityCodeFixVerifier`. This helper method is unnecessary if you fix the definition of `VerifyCS` above.